.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: node_js
+
 notifications:
   email: false
+
 before_script:
   - npm prune
-after_success:
-  - npm run semantic-release
-branches:
-  except:
-    - /^v\d+\.\d+\.\d+$/
+
+deploy:
+  provider: script
```

Comment:
Travis CI's docs say this:

>Do note that custom scripts are still considered an experimental feature.

Any idea what they mean by this? Is it a bad idea for us to depend on this right now? 🤔

Reply:
It's not marked in their Beta features. And seems to be quite commonly used as-well as recommended by semantic-release.
I'd be comfortable relying on the feature.